packages/pagination/src/elements/Pagination.spec.js

```diff
@@ -69,184 +65,172 @@ describe('Pagination', () => {
         return props;
       };
 
-      const wrapper = mountWithTheme(
+      const { container } = render(
         <BasicExample currentPage={1} totalPages={10} transformPageProps={transformPageProps} />
       );
 
-      wrapper
-        .find('ul')
-        .children()
-        .forEach(child => {
-          expect(child).toHaveProp('data-test-id', CUSTOM_PROP_VALUE);
-        });
+      for (let index = 0; index < container.firstChild.children.length; index++) {
+        const child = container.firstChild.children[index];
+
+        expect(child).toHaveAttribute('data-test-id', CUSTOM_PROP_VALUE);
```

Comment:
You could just spread the children into an array and keep the similar forEach approach

```js
[...container.firstChild.children].forEach(child => {
  expect(child).toHaveAttribute('data-test-id', CUSTOM_PROP_VALUE);
});
```